charts/substra-backend/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [22.8.5] - 2023-10-09
+
+## Added
+
+- `bitnami/common` dependency added 
+
+## Changed
+
+- `redis` subchart version incremented to `18.1.2`
+- `postgresql` subchart version incremented to `13.1.0`
```

Comment:
This is a breaking change, could you
 - upgrade the chart version to `23.0.0`;
 - add `BREAKING: ` in front of this;
 - and add an entry in `charts/substra-backend/UPGRADE.md`?

Reply:
Sure of course!